src/OrchardCore/OrchardCore.ResourceManagement.Abstractions/ResourceDefinition.cs

```diff
@@ -98,7 +99,7 @@ public ResourceDefinition SetUrl(string url, string urlDebug)
         {
             if (String.IsNullOrEmpty(url))
             {
-                throw new ArgumentNullException("url");
```

Comment:
here we have many small methods that could not be inlined when they had `throw`, now letting JIT to decide